app/build.gradle

```diff
@@ -54,10 +54,10 @@ android {
     kotlinOptions {
         jvmTarget = '11'
     }
-
-    lintOptions {
-        disable "JvmStaticProvidesInObjectDetector", "FieldSiteTargetOnQualifierAnnotation", "ModuleCompanionObjects", "ModuleCompanionObjectsNotInModuleParent"
+    lint {
+        disable 'JvmStaticProvidesInObjectDetector', 'FieldSiteTargetOnQualifierAnnotation', 'ModuleCompanionObjects', 'ModuleCompanionObjectsNotInModuleParent'
     }
+
```

Comment:
Ez miért változott?

Reply:
Mert az AGP verzió is megváltozott, egy ajánlott módosítás volt, amit az Android Studio dobott fel.